modules/raw-rsa-keyring-browser/test/get_import_options.test.ts

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may not use
+ * this file except in compliance with the License. A copy of the License is
+ * located at
+ *
+ *     http://aws.amazon.com/apache2.0/
+ *
+ * or in the "license" file accompanying this file. This file is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied. See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* eslint-env mocha */
+
+import * as chai from 'chai'
+import chaiAsPromised from 'chai-as-promised'
+import 'mocha'
+import {
+  flattenMixedCryptoKey,
+  verify
+} from '../src/get_import_options'
+
+chai.use(chaiAsPromised)
+const { expect } = chai
+
+describe('flattenMixedCryptoKey', () => {
+  it('Check for early return (Postcondition): nothing is an empty array.', () => {
```

Comment:
This took me a couple of tries to read: perhaps "empty inputs should return an empty array"?

Reply:
updated name